recipes/icqsol/meta.yaml

```diff
@@ -0,0 +1,39 @@
+package:
+  name: icqsol
+  version: "0.3.20"
+
+about:
+  home: 'https://github.com/pletzer/icqsol'
+  summary: 'A collection of utilities for constructing complex geometries from primitive shapes'
+  license: MIT
+
+source:
+  git_url: https://github.com/pletzer/icqsol.git
+  git_rev: 05b229c
+
+build:
+  number: 0
+
+test:
+  # Python imports
+  imports:
+    - csg
+    - icqsol
+    - numpy
+    - triangle
+    - vtk
+
+requirements:
+  build:
+    - cmake
+    - numpy
+    - python
+    - pycsg ==0.3.12
```

Comment:
Ok, my understanding is that if these versions are pinned like this and if someone creates a new version of pycsg, when this version of icqsol is installed, the pinned version of pycsg (i.e., 0.3.12) will come with it.  If I understand correctly, I think we should leave these pinned.  I've tested this on OS X and Linux.  Thanks!